src/pages/commits/BuildVariantAccordion.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Disclaimer } from "@leafygreen-ui/typography";
+
+const { gray } = uiColors;
+
+interface Props {
+  buildVariants: {
+    displayName: string;
+  }[];
+}
+export const BuildVariantAccordion: React.FC<Props> = ({ buildVariants }) => (
+  <ColumnContainer>
+    {buildVariants.map(({ displayName }) => (
+      <AccordionContainer key={displayName}>{displayName}</AccordionContainer>
```

Comment:
We have an [Accordion component](https://github.com/evergreen-ci/spruce/blob/ee5c3c4fe6d8af169696475e0e168249703cc5c9/src/components/Accordion/index.tsx) that would be useful here. I know you aren't rendering any tasks yet but this will likely come in handy when you do.